### PR TITLE
Add py-storm@0.23 and update URL

### PR DESCRIPTION
**Pull request by iarspider**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-storm/package.py [hunk 1]
```diff
@@ -11,6 +11,21 @@ class PyStorm(PythonPackage):
...
+    def url_for_version(self, version):
+        url = self.url.split('/')
+        url[5] = '{0}'
+        if version >= Version('0.20'):
+            url[-1] = '/storm-{0}.tar.bz2'
+        else:
+            url[-1] = '/storm-{0}.tar.gz'
+        
+        url = '/'.join(url).format(version)
+        return url
```
Comment (michaelkuhn) [inline var/spack/repos/builtin/packages/py-storm/package.py]: Why not just update `url`? 0.20 is the oldest version in Spack anyway. Moreover, if I don't have mixed anything up, all versions are available as `tar.bz2` according to https://launchpad.net/storm/+download.

**Review by michaelkuhn — APPROVED** [review]

**Comment by michaelkuhn** [comment]
@alalazo Looks like one of the Travis checks is still hanging, so I can't merge (newer PRs seem to be fine, though). Anything else we can do?

**Comment by alalazo** [comment]
@michaelkuhn Neither can I. Can you rebase on latest `develop` or something similar? A force push might take this PR out of the loop it's in with notifications.

**Comment by michaelkuhn** [comment]
@iarspider That did the trick, thanks!

**Pull request merged**